code/mob/who.dm

```diff
@@ -124,4 +124,5 @@
 	if(!usr.client.holder)
 		logTheThing("admin", usr, null, "used adminwho and saw [adwnum] admins.")
 		logTheThing("diary", usr, null, "used adminwho and saw [adwnum] admins.", "admin")
-		message_admins("<span class='internal'>[key_name(usr)] used adminwho and saw [adwnum] admins.</span>")
+		if(usr.client.holder.adminwho_alerts)
```

Comment:
Uh, I don't think that's gonna work. See the `if` block that it's in...

Reply:
Fixed and tested on local server with 2 accounts connected, one of them being a regular player one, works fine now.